services/src/main/java/org/broadleafcommerce/payment/service/gateway/DefaultPayPalCheckoutTransactionService.java

```diff
@@ -150,10 +161,14 @@ public PaymentResponse capture(PaymentRequest paymentRequest) {
             Capture capture = capturePayment(auth, paymentRequest);
             Amount amount = capture.getAmount();
 
+            String captureTime = capture.getCreateTime();
+
             paymentResponse
                     .successful(true)
                     .rawResponse(capture.toJSON())
+                    .gatewayTransactionId(capture.getId())
                     .responseMap(MessageConstants.CAPTUREID, capture.getId())
+                    .dateRecorded(captureTime != null ? Instant.parse(captureTime) : Instant.now())
                     .amount(MonetaryUtils.toAmount(amount.getTotal(), amount.getCurrency()));
         } catch (PaymentException e) {
             processException(e, paymentResponse, paymentRequest);
```

Comment:
When processing the exception, use `PaymentResponse#message`, instead of `ERROR_MESSAGE` in the responseMap.